src/App.js

```diff
@@ -1,10 +1,12 @@
-import React from 'react';
+import React from "react";
+import MainPage from "./components/mainpage/componens.mainpage.mainpage";
 
 function App() {
   return (
-    <div > 14기 프론트엔드 짱 =_=
+    <div>
+      <MainPage />
     </div>
```

Comment:
의미없는 `div`는 보통 `React.Fragment`로 대체하는 것이 좋습니다. `div`는 DOM Tree에 포함되지만 `Fragment`는 코드에만 존재할 뿐 실제 DOM tree에 노드를 추가하지 않기 때문이죠.
```suggestion
    <Fragment>
      <MainPage />
    </Fragment>
```

Reply:
와웅 오늘도 배워갑니다...